### DISCUSSION TITLE
Should we make cargo-component public?

### DISCUSSION THREAD

**Discussion by peterhuene** [bytecodealliance/SIG-Registries#18] [General]
I wanted to start a discussion on whether or not [`cargo-component`](https://github.com/bytecodealliance/cargo-component) should be made a public repo.

While it lacks any integration into the registry work while we design the registry APIs, it is useful on its own for creating single components that we can test Wasmtime and other tools with.

Keeping it private might help us iterate on the registry integration before there's public announcements, though.

What are your thoughts?

**Comment by ricochet** [comment]
I think it's known that we're hacking on registry API so I'm not too worried about making changes that would require a public announcement. I vote we mark it experimental and make it public

**Reply by radu-matei** [reply]
We could push a private fork where we work on the registry API implementation.

**Comment by peterhuene** [comment]
I think that it makes sense to make the repo public and continue any registry work that we're iterating on to a private fork.

If @tschneidereit or @squillace or someone else with admin rights over the repository agrees, can we make the cargo-compoment repo public (I don't have access rights to do so)? Thanks!
